src/main/java/no/ssb/guardian/maskinporten/MaskinportenAccessTokenController.java

```diff
@@ -183,6 +183,11 @@ public HttpResponse<JsonError> clientRequestError(HttpRequest request, Maskinpor
         return error(request, e, e.getHttpStatus(), e.getMessage());
     }
 
+    @Error
+    public HttpResponse<JsonError> runtimeError(HttpRequest request, RuntimeException e) {
+        metrics.incrementClientError("runtime-exception");
```

Comment:
maybe instead use metrics.incrementServerError for runtime exceptions?

Reply:
Fixed